### ISSUE TITLE
[Design] 526: Fully Developed Claim screen - UX Enhancements

### ISSUE THREAD

**Issue by kevinstachura** [department-of-veterans-affairs/va.gov-team#19477]
## User Story or Problem Statement
As a veteran, I need a clearer picture on whether I should select 'Fully Developed Claim' or not when submitting my 526 disability claim online.

### Why we're asking this question

Recently, our backend devs [collected some data on drop-off points for the 526 form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/15689#issuecomment-769944735). Among them was a fair amount of drop-offs on the fully developed claim page. After assessing whether or not this was tech or experience design-related, we determined it was likely to be a design question that requires some research and UI edits.

Current state of screen:

<img width="769" alt="Screen Shot 2021-02-03 at 3 26 04 PM" src="https://user-images.githubusercontent.com/54544300/106814466-6eee8e80-6640-11eb-8a31-aee7e669267c.png">

Some specific questions to consider:

- What truly qualifies for FDC (How does the Veteran know that VBA has everything they need to decide the claim)? 
- How does it benefit me? Does my claim get processed more quickly?
- Will I be ‘penalized’ if I’m wrong? (i.e. what are the ramifications of actions on this page?) 
- How do I know the requirements for FDC?
- Can we make this more generic ('Have you uploaded all necessary documentation for your claim?')?

## Tasks
- [x] Chat with VBA to understand business requirements around FDC
- [x] Make proposed UI design changes
- [ ] Add this proposed change to list of enhancements to test in a general 526 usability initiative

**Comment by sshein** [comment]
I'm interested this so I'm commenting so I see what ends up being recommended :)

**Comment by kevinstachura** [comment]
After chatting with Paul Schute and the BDD team, it was clear that surfacing the information specific to FDC is not necessary in the flow. We have created these solutions:

1. For BDD, it is completely unnecessary to ask about FDC. It does not affect their claim processing and thus **we will not show this screen at all**.
2. For regular 526 filings, it is still relevant and necessary. However, the way we are asking is likely complex and possibly intimidating, so we should simply be asking whether or not the veteran has submitted all the documentation necessary for their claim. Here is a simplified version of what we could do to make this process easier on the user:

https://vsateams.invisionapp.com/share/9Y10APGURJSV#/445684149_Fully_Developed_Claim_-_Non-FDC_Version_-_Concept_

This would be a great baseline to test out and may simply eliminate a lot of questions the user may have in qualifying for FDC, consequences, etc. That said, Christian and I talked about taking it a step further by testing out a version with more contextual information (yet to be determined by a call with VBA) on how they might proceed:

https://vsateams.invisionapp.com/share/9Y10APGURJSV#/445677525_Fully_Developed_Claim_-_Non-FDC_Version_-_Concept

Once we get content details from VBA, we could potentially test both versions and see if the simplified version is enough, or if the contextual version provides more value to users in that process.

@sshein let Christian and I know if you have any thoughts on this approach - thank you!

Closing this ticket as the scoped acceptance criteria has been met.

**Comment by sshein** [comment]
@kevinstachura definitely much simpler and better, but I think the copy might still confused veterans a little - they might be like "um, I'm not sure if I've uploaded everything I'm supposed to, how would I know?"... maybe the question can be more clear, like "Are you waiting on any documentation that you will need to submit later" with a Yes/No.

**Comment by sshein** [comment]
Are you making another ticket to nail down the design/copy? may be worth tagging Danielle or Beth quickly to get their thoughts

**Comment by kevinstachura** [comment]
@sshein this:

>Are you waiting on any documentation that you will need to submit later?

Is certainly _clearer_. It's representative of how we want to approach this, which is "did you submit everything that **you** have to support this claim, or no?" rather than the concept of "did you submit everything VA **needs** to make a decision on this?" which is almost impossible for them to answer, and pretty much impossible info for us to _give_, given all the nuances case-by-case of what VA might need. 

I only hesitate on using the term 'waiting' here - because it's rather specific to say 'waiting' on something. Might be that they still need to even request things in the first place ('I'm gonna get the ball rolling on this filing and then req my DD214 and add that later'), or it is possible they may have the documentation, but not on hand ('oops - I forgot that on a thumb drive somewhere'). 

Did you think the contextual information on this dropdown answered that question? In the rough draft copy I have it say something about waiting on documentation:
> 'If you have more evidence or are waiting on some documentation, you can send it to us in a few ways:' 

which I thought might suffice. Or do you think they won't even try to click it, if the more general question wording throws them off? 🤔 And to answer your question, we can definitely ping Danielle/Beth before testing but **after** getting the content requirements from VBA for that dropdown - so we have the proper content to wordsmith. 🙂

**Comment by sshein** [comment]
exactly this 💯 : "It's representative of how we want to approach this, **which is "did you submit everything that you have to support this claim, or no?" rather than the concept of "did you submit everything VA needs to make a decision on this?" which is almost impossible for them to answer,** and pretty much impossible info for us to give, given all the nuances case-by-case of what VA might need." 

Maybe it's just a matter of wordsmithing the sentence so the above is more clear. I agree "waiting" may not be the solution.

My hunch is they won't even click the "additional info" component if the initial question is too confusing - or rather that we should try to make the initial question more clear and not rely on the additional info component

**Comment by sshein** [comment]
Maybe "Is there additional documentation you don't have on hand now, but would like to submit later?" or something to that effect
